src/app/find-help/view-offer-material-aid/view-offer-material-aid.component.ts

```diff
@@ -31,6 +31,10 @@ export class ViewOfferMaterialAidComponent implements OnInit {
       .catch((e) => console.error(e));
   }
 
+  getListUrl() {
```

Comment:
Refactor suggestion
`
getListUrl(url: string): string {
   return url.replace(/\/[^/]+$/, '');
 }
`

And put it in any util.ts or sth